scripts/core/enqueue.yml

```diff
@@ -1044,8 +1044,8 @@ params:
             Input can not be a nullptr.
             TODO: change to allow nullptr. 
 returns:      
-    - $X_RESULT_INVALID_QUEUE
-    - $X_RESULT_INVALID_EVENT
-    - $X_RESULT_INVALID_MEM_OBJECT
-    - $X_RESULT_OUT_OF_HOST_MEMORY
-    - $X_RESULT_OUT_OF_RESOURCES
\ No newline at end of file
+    - $X_RESULT_ERROR_INVALID_QUEUE
+    - $X_RESULT_ERROR_INVALID_EVENT
+    - $X_RESULT_ERROR_INVALID_MEM_OBJECT
+    - $X_RESULT_ERROR_OUT_OF_HOST_MEMORY
+    - $X_RESULT_ERROR_OUT_OF_RESOURCES
```

Comment:
Please add a trailing new line to the end of this file.

Reply:
:+1: